### ISSUE TITLE
Bug File not found: '/home/runner/work/_actions/JamesIves/fetch-api-data-action/1.0.11/lib/main.js

### ISSUE THREAD

**Issue by Valiums** [JamesIves/fetch-api-data-action#141]
First of all, thanks for this action @JamesIves.

Seems to be some files missing in your latest version if I compare the 1.0.9 version with this one, there is no lib folder provided

(1.0.9)
![image](https://user-images.githubusercontent.com/14872072/88745983-5c3b5000-d119-11ea-93d3-bc5e745da58c.png)

vs 

(1.011)
![image](https://user-images.githubusercontent.com/14872072/88746005-678e7b80-d119-11ea-8ba0-e73ce3430a84.png)

And here's the error I am getting 

![image](https://user-images.githubusercontent.com/14872072/88746029-7b39e200-d119-11ea-8594-973bcf27aec5.png)

And just in case, here's my action

```

name: Cron jobs

on:
  schedule:
    - cron: 0 8 * * *

jobs:
    clean:
    runs-on: ubuntu-latest
    steps:
      - name: Fetch API Data 📦
        uses: JamesIves/fetch-api-data-action@releases/v1
        with:
          ENDPOINT: https://myapi.com
          CONFIGURATION: '{ "method": "GET", "headers": {"Authorization": "${{ secrets.MY_SECRET }}"} }'


```

Maybe I am just doing something wrong. Hopefully this will help if this is a real issue.

**Comment by JamesIves** [comment]
Definitely an issue, thanks for reporting! I've fixed this in `1.0.15`: https://github.com/JamesIves/fetch-api-data-action/tree/1.0.15

Must have based the previous release on a different branch when I was adding the GitHub registry publishing pipeline.

**Comment by Valiums** [comment]
@JamesIves awesome thanks!
